### PR TITLE
Add Entry: Java Type-Conversion

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

From several entries on the Target Keyword sheet. Entry for Java type-conversion

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/type-conversion/type-conversion.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+Title: 'Type Conversion'
+Description: 'Type conversion is the process where a value of one data type is converted to a value of another data type.'
```
Comment (KTom101) [inline content/java/concepts/type-conversion/type-conversion.md:3]:
```suggestion
Description: 'Type conversion is the process of converting a value of one data type into another data type.'
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@SSwiniarski just finished with the first review of this.

**Comment by SSwiniarski** [comment]
@KTom101, implemented your suggestions.

**Review by yangc95 — COMMENTED** [review]
2️⃣ nd review fin.

**Comment by SSwiniarski** [comment]
@yangc95, I committed your suggestions.

**Review by KTom101 — APPROVED** [review]
@SSwiniarski Looks good 👍

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@SSwiniarski live link: https://www.codecademy.com/resources/docs/java/type-conversion

ty @KTom101 🥳

**Pull request merged**